### ISSUE TITLE
Calling mdb_test_sarg on unknown type.  Add code to mdb_test_sarg() for type 7

### ISSUE THREAD

**Issue by marnunez** [mdbtools/mdbtools#105]
I'm running a pyodbc query through mdbtools on an old mdb database. Stuff like:
```SQL
select COL1, COL2 from TABLE1
```

runs smoothly. But as soon as I try adding a simple filter such as:
```SQL
select COL1, COL2 from TABLE1 where COL1=200
```
it throws `Calling mdb_test_sarg on unknown type.  Add code to mdb_test_sarg() for type 7`
and the rows are returned unfiltered.

Debugging by adding

```
[ODBC]
Trace=yes
TraceFile=/tmp/odbc.log
```

into /etc/odbcinst.ini, in the log I find:

```
[ODBC][18832][1481316566.824810][SQLDescribeCol.c][497]
		Exit:[SQL_SUCCESS]                
			Column Name = [COL1]                
			Data Type = 0x7ffeb5b5aeda -> 8                
			Column Size = 0x7ffeb5b5aee0 -> 8                
			Decimal Digits = 0x7ffeb5b5aedc -> 0                
			Nullable = 0x7ffeb5b5aede -> 0
```

Any ideas? Am I missing something here?

**Comment by rhurlin** [comment]
I just triet with mdbtools version 0.7.1 on FreeBSD and it works for me.

Is COL1 of TABLE1 really a numeric field and not a textual one, only containing numbers?

Is your installed version of mdbtools an official package of your distro?

**Comment by markord** [comment]
FYI, type 7 is DOUBLE and mdb_test_sarg() (in libmdb/sarg.c) is not implemented.

I've come across the same situation, but with a MEMO field.

**Comment by evanmiller** [comment]
@nyalldawson Any interest in tackling this one?

**Comment by evanmiller** [comment]
A fix is now included in #162 – closing issue.
